test/conversion.test.js

```diff
@@ -27,6 +27,16 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	it('[Github: #1]: not throw an error for having $ before method', function (done) {
+		convert({
+			type: 'string',
+			data: `curl -X $'POST' 'https://example.com.br/login.html'`
+		}, function (err, result) {
+			expect(result.result).to.equal(true);
```

Comment:
Let's also check for the actual request.method being POST. The other test that's added is a unit test. We should test the e2e flow here - which is not limited to `result` being true.